bootstrap.sh

```diff
@@ -0,0 +1,108 @@
+#!/bin/bash
+
+set -eux -o pipefail
+
+TLA=${LU_TLA}
+TOK=${LU_TOKEN}
+
+# determine distro
+
+if [[ -f /etc/lsb-release && $(grep -Eo 'DISTRIB_ID=Ubuntu' /etc/lsb-release) ]]; then
+    DISTRO="Ubuntu"
+fi
+
+if [[ -f /etc/redhat-release && $(grep -Eo 'release 7\.' /etc/redhat-release) ]]; then
+    DISTRO="RHEL7"
+fi
+
+if [[ -f /etc/redhat-release && $(grep -Eo 'release 8\.' /etc/redhat-release) ]]; then
+    DISTRO="RHEL8"
+fi
+
+if [[ x$DISTRO = x ]]; then
+    echo "error: distribution not supported"
+    exit 1
+fi
```

Comment:
I have this already coded up in ./utils.sh

Reply:
not sure how to use it though because bootstrap.sh will be the only script available when this distro needs to be resolved. utils.sh won't be available.